variables.tf

```diff
@@ -37,6 +37,10 @@ variable "availability_zones" {
 
 variable "bastion_ami" {}
 
+variable "bastion_ebs_optimized" {
+  default = true
```

Comment:
I would consider defaulting this to `false`. It works well in the default case within the context of this module (converting `bastion_instance_type` from T2 to T3), but may otherwise cause challenges for other setups using instance types without EBS optimized support.